src/main/java/uk/gov/companieshouse/model/Reason.java

```diff
@@ -26,23 +28,41 @@ public void setOther(OtherReason other) {
         this.other = other;
     }
 
+    public IllnessReason getIllnessReason() { return this.illnessReason; }
```

Comment:
Please keep the formatting consistent

Reply:
May be worth checking that Intellij styles are set up correctly as I know Mo and Gareth are new to it.

Reply:
Done. 

I will set IntelliJ style as soon as possible